### PR TITLE
DEV-2302 Extend mimetypes with all existing extensions

### PR DESCRIPTION
**Pull request by maartends**
All existing file extensions and their mime types were added to the mapping. The strings for the "Library of Congress Recommended Formats Statements" were replaced with constants.
Tests were extended to cover all cases.

### CODE REVIEW

File: app/helpers/bag.py [hunk 1]
```diff
@@ -50,50 +50,83 @@
...
+    ".mp4": "video/mp4",
+    ".mpeg": "video/mpeg",
     ".mpg": "video/mpeg",
+    ".mts": "video/MP2T",
+    ".mxf": "application/mxf",
     ".ogg": "audio/ogg",
-    ".zip": "application/zip",
+    ".pdf": "application/pdf",
+    ".png": "image/png",
+    ".psb": "image/vnd.adobe.photoshop",
```
Comment (spacid) [inline app/helpers/bag.py:89]: Hmm, does it need to be "image/vnd.adobe.photoshop"? I got the mimetype value from MediaHaven but it is not the correct one?

Comment (maartends) [reply app/helpers/bag.py:89]: Seems that it should be `image/vnd.adobe.photoshop`:
- https://www.iana.org/assignments/media-types/media-types.xhtml
- https://mimetype.io/image/vnd.adobe.photoshop.

Comment (spacid) [reply app/helpers/bag.py:89]: `image/vnd.adobe.photoshop` used in  `.psd`. Different file though but `image/vnd.adobe.photoshop` is fine by me.

File: app/helpers/bag.py [hunk 2]
```diff
@@ -50,50 +50,83 @@
...
-    "application/mxf": "Video - File-based and Physical Media",
-    "application/vnd.adobe.photoshop": "Photographs - Digital",
-    "video/x-matroska": "Video - File-based and Physical Media",
-    "video/x-msvideo": "Video - File-based and Physical Media",
+    "application/mxf": VIDEO_FILE_BASED,
+    "application/pdf": TEXTUAL_DIGITAL,
+    "application/zip": COLLECTION,
+    "audio/ac3": AUDIO_MEDIA_INDEPENDENT,
+    "audio/aiff": AUDIO_MEDIA_INDEPENDENT,
+    "audio/mp4": VIDEO_FILE_BASED,
```
Comment (spacid) [inline app/helpers/bag.py]: Moet dit `AUDIO_MEDIA_INDEPENDENT` zijn?

**Review by spacid — CHANGES_REQUESTED** [review]

File: app/helpers/bag.py (hunk 2 above)
Comment (maartends) [reply app/helpers/bag.py]: Correct. Fixed.

**Review by spacid — APPROVED** [review]

**Pull request merged**